src/components/top-tab-bar/TopTabBar.tsx

```diff
@@ -42,8 +42,14 @@ const createTabBarStyles = (themeColors: ThemeColors) =>
   StyleSheet.create({
     tabBarContainer: {
       flexDirection: 'row',
+      marginBottom: -4,
       paddingBottom: 4,
```

Comment:
The padding is needed to display the tab indicator bc content outside the component is cut out, but the margin will remove the unnecessary space taken up by the padding